i18n/it.json

```diff
@@ -143,13 +143,13 @@
     "%s requires the PATH to end in a \"%s\". Please update the PATH in your index.php.":
         "%s necessita che PATH termini con \"%s\". Aggiorna la variabile PATH nel tuo index.php.",
     "Decrypt":
-        "Decrypt",
+        "Decifra",
     "Enter password":
         "Inserisci la password",
-    "Loading…": "Loading…",
-    "Decrypting paste…": "Decrypting paste…",
-    "Preparing new paste…": "Preparing new paste…",
+    "Loading…": "Carico…",
+    "Decrypting paste…": "Decifro il messaggio…",
+    "Preparing new paste…": "Preparo il nuovo messaggio…",
     "In case this message never disappears please have a look at <a href=\"https://github.com/PrivateBin/PrivateBin/wiki/FAQ#why-does-not-the-loading-message-go-away\">this FAQ for information to troubleshoot</a>.":
-        "Nel caso questo messaggio non scompaia, controlla questa <a href=\"https://github.com/PrivateBin/PrivateBin/wiki/FAQ#why-does-not-the-loading-message-go-away\"> FAQ</a> per trovare informazioni su come risolvere il problema (in Inglese).",
-    "+++ no paste text +++": "+++ no paste text +++"
+        "Nel caso questo messaggio non scompaia, controlla questa <a href=\"https://github.com/PrivateBin/PrivateBin/wiki/FAQ#why-does-not-the-loading-message-go-away\">FAQ</a> per trovare informazioni su come risolvere il problema.",
```

Comment:
I totally agree, that was a misunderstanding: I canceled the translation "(in Inglese)" because I couldn't find the original "(in English)" and I thought you deleted it, but that part was only present in the original untranslated sentence. It was deleted by me when I first translated it, lol...